src/components/Toast.tsx

```diff
@@ -37,20 +35,24 @@ const Toast = () => {
           left: [null, null, "32px"],
           right: [null, null, "32px"],
           top: "32px",
-        }
+        },
       } }
     >
       <Alert
         iconMapping={ {
           error: <CloseCircleFill />,
-          success: <CheckCircleIcon />
+          success: <CheckCircleIcon />,
         } }
         onClose={ handleClose }
         severity={ severity }
         sx={ {
           "&.MuiAlert-root": {
             ".MuiAlert-message": {
-              paddingRight: ["8px", "16px", "48px"]
+              paddingRight: [
```

Comment:
Going forward, I think we should use either p, pr, pl... or padding, paddingRight, paddingLeft.... My vote is for the former for brevity.

Reply:
We can definitely talk about it, these are the cons of us having the flexibility to do it multiple ways :/